docs/pipeline.rst

```diff
@@ -143,3 +143,41 @@ DataNode
 --------
 .. autoclass:: nvidia.dali.pipeline.DataNode
    :members:
+
+Pipeline Debug Mode (experimental)
+------------------
+
+Pipeline can be run in debug mode by replacing :meth:`@nvidia.dali.pipeline_def <nvidia.dali.pipeline_def>` decorator with its experimental variant
+``@nvidia.dali.experimental.pipeline_def`` and setting parameter ``debug`` to True. It allows you
+to access and modify data inside of the pipeline execution graph as well as use non-DALI data types
```

Comment:
Specifying 
``` :meth:`@nvidia.dali.pipeline_def <nvidia.dali.pipeline_def>` ``` and then ``` ``@nvidia.dali.experimental.pipeline_def`` ``` makes the former clickable (and actually redirects to the doc of the `pipeline_def`) and the latter non-clickable and formatted differently. May we make it consistent?

Reply:
```suggestion
to access and modify data inside the pipeline execution graph, as well as use non-DALI data types
```

Reply:
done